force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -270,12 +343,19 @@ public with sharing class FlowControlLoopsCollections {
      * @return The position of the word in the list as a string or -1 if the word was not found.
      */
     public static Integer findWordInList(String wordToFind, List<String> words) {
-        // The variable to store the index of the word
-
-        // Loop through the list of words
-            // If the current word is the word to find, exit the loop
-
-        return null; // Replace null with the variable you used to store the result
+        // Don't need this class do we?  We could use line below in calling code, I think.
+        Integer wordToFindLocale = words.indexOf(wordToFind);
+
+        // No interation needed if we use .indexOf but could do it this way.
+        // Integer wordToFindLocale = -1;
+        // for( Integer i = 0; i < words.size(); i++ ) {
+        //     if ( words[i] == wordToFind ) {
+        //         wordToFindLocale = i;
+        //         break;
+        //     }
+        // }
```

Comment:
They are both correct and two different ways of doing it. Nice you were able to find both ways